pom.xml

```
@@ -485,12 +479,15 @@
                 <configuration>
                     <source>${java.version}</source>
                     <target>${java.version}</target>
+                    <compilerArgs>
+                        <arg>-Xlint:all,-path,-options,-deprecation</arg>
```

```suggestion
                        <arg>-Xlint:all,-path,-options,-deprecation</arg>
                        <arg>-Werror</arg>
```
If possible remove `-deprecation` here and use `@SuppressWarnings("...")` in source code